roles/kubernetes-apps/helm/tasks/install_docker.yml

```diff
@@ -1,8 +0,0 @@
----
```

Comment:
Why do we need to remove this file?

Reply:
Helm v3 is just a go static binary, so I don't see why someone would want to pull a container for it :)

Reply:
Ok for you @oomichi ?

Reply:
Yeah, I cannot find any places which use this file.
It seems fine to remove it for me.